packages/amplify-category-api/src/category-utils/show-auth-acm.ts

```diff
@@ -0,0 +1,148 @@
+import {
+  AccessControlMatrix,
+  AuthRule,
+  ModelOperation,
+  MODEL_OPERATIONS,
+  DEFAULT_GROUPS_FIELD,
+  DEFAULT_GROUP_CLAIM,
+  DEFAULT_OWNER_FIELD,
+  getAuthDirectiveRules,
+} from '@aws-amplify/graphql-auth-transformer';
+import { $TSContext, stateManager, pathManager, FeatureFlags } from '@aws-amplify/amplify-cli-core';
+import { parse, ObjectTypeDefinitionNode, DirectiveNode, FieldDefinitionNode } from 'graphql';
+import { printer } from '@aws-amplify/amplify-prompts';
+import { DirectiveWrapper } from '@aws-amplify/graphql-transformer-core';
+import { readProjectSchema } from 'graphql-transformer-core';
+import { getTransformerVersion } from '../graphql-transformer';
+import * as path from 'path';
+
+export const showApiAuthAcm = async (context: $TSContext, modelName: string): Promise<void> => {
+  const providerPlugin = await import(context.amplify.getProviderPlugins(context)?.awscloudformation);
+  const transformerVersion = await getTransformerVersion(context);
+
+  if (transformerVersion < 2) {
+    printer.error('This command requires version two or greater of the GraphQL transformer.');
+    return;
+  }
+
+  const apiNames = Object.entries(stateManager.getMeta()?.api || {})
```

Comment:
I see this multi-line logic to fetch the name of the API added. Wonder if there already is a common utility method to do this. Otherwise, ignore.